DDDEastMidlandsLimited/dddem-web
# Bug: Improve Jest Tests For Footer #259

Issue's link https://github.com/DDDEastMidlandsLimited/dddem-web/issues/259

Fixes:
- [X] I've provided a manual mock for FontAwesomeIcon and imported into Footer.test.js

How to test:
Run `npm run test `
you should see all tests passed and 0 errors into the terminal
